src/main/java/uk/ac/ebi/eva/pipeline/io/VepProcess.java

```diff
@@ -110,7 +110,8 @@ public void open() throws ItemStreamException {
                 "-o", "STDOUT",
                 "--force_overwrite",
                 "--offline",
-                "--everything"
+                "--everything",
+                "--format", "ensembl"
```

Comment:
Do older versions of VEP just ignore this parameter? Please tell me they don't throw an error and fail :pray: